comiccrawler/download_manager.py

```diff
@@ -190,21 +137,43 @@ def start_analyze(self, mission):
 			print(
 				"Invalid state to analyze: {state}".format(state=mission.state))
 			return
-		init_episode(mission)
-		thread = Worker(analyze).start(mission)
+		def do_analyze():
+			with load_episodes(mission):
+				analyze(mission)
```

Comment:
Is it possible to hook `select_episodes` in `main_window` to here? We need a way to communicate with GUI thread and get the response.